xbmc/music/windows/GUIWindowVisualisation.cpp

```diff
@@ -56,7 +57,7 @@ bool CGUIWindowVisualisation::OnAction(const CAction &action)
     // If confirm channel switch is active, channel preview is currently shown
     // and the button that caused this action matches (global) action "Select" (OK)
     // switch to the channel currently displayed within the preview.
-    CServiceBroker::GetPVRManager().ChannelPreviewSelect();
+    CServiceBroker::GetPVRManager().GUIActions()->GetChannelPreview().SelectChannel();
```

Comment:
Makes perfect sense. I found no good solution so far. But this is definitely on my radar for another day (PR).